webpack/scenes/RedHatRepositories/RedHatRepositoriesPage.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint import/no-unresolved: [2, { ignore: [foremanReact/*] }] */
+/* eslint-disable import/no-unresolved */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'react-bootstrap';
+import { Button } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PermissionDenied from 'foremanReact/components/PermissionDenied';
+import { LoadingState } from '../../move_to_pf/LoadingState';
+import { createEnabledRepoParams } from '../../redux/actions/RedHatRepositories/enabled';
+import SearchBar from './components/SearchBar';
+import RecommendedRepositorySetsToggler from './components/RecommendedRepositorySetsToggler';
+import { getSetsComponent, getEnabledComponent } from './helpers';
+import api from '../../services/api';
+
+class RedHatRepositoriesPage extends Component {
+  componentDidMount() {
+    this.loadData();
+  }
+
+  loadData() {
+    this.props.loadEnabledRepos();
+    this.props.loadRepositorySets({ search: { filters: ['rpm'] } });
+  }
+
+  render() {
+    const { enabledRepositories, repositorySets } = this.props;
+    const { repoParams } = createEnabledRepoParams(enabledRepositories);
+
+    if (repositorySets.missingPermissions && repositorySets.missingPermissions.length > 0) {
```

Comment:
you can use `lodash.isEmpty`:
```if ( !isEmpty(repositorySets.missingPermissions) )```
and
```if ( !isEmpty(enabledRepositories.missingPermissions) )```
looks cleaner

Reply:
@laviro Cool, didn't know that one :)